tests/integration/tests/test_control_plane_taints.py

```diff
@@ -6,28 +6,19 @@
 from typing import List
 
 import pytest
-import yaml
-from test_util import harness, util
+from test_util import config, harness, util
 
 LOG = logging.getLogger(__name__)
 
 
 @pytest.mark.node_count(1)
-@pytest.mark.disable_k8s_bootstrapping()
+@pytest.mark.bootstrap_config(
+    config.MANIFESTS_DIR / "bootstrap-control-plane-taints.yaml"
```

Comment:
maybe worth making this not a path, but the contents of the config itself. then callers can do `(config.MANIFESTS_DIR / "bootstrap-control-plane-taints.yaml").read_text()` or specify the config directly

Reply:
good point, updated